jme3-xr/src/main/java/com/jme3/input/xr/XrHmd.java

```diff
@@ -0,0 +1,94 @@
+package com.jme3.input.xr;
+
+import java.util.ArrayList;
+
+import com.jme3.app.SimpleApplication;
+import com.jme3.math.Quaternion;
+import com.jme3.math.Vector3f;
+import com.jme3.system.AppSettings;
+import com.jme3.system.lwjgl.LwjglWindowXr;
+
+public class XrHmd
+{
+	public static final float DEFAULT_X_DIST = 0.4f;
+	public static final float DEFAULT_X_ROT = 0.028f;
```

Comment:
What is DEFAULT_X_ROT for? It looks to be 1.6 degrees which is a tiny amount

Reply:
Yes, it is Rotation.
Without this value, far objects looks like they appear twice.